### PR TITLE
Add Caisse Epargne bank (FR)

### PR DESCRIPTION
**Pull request by chatainsim**
Add Caisse Epargne bank (FR)

### CODE REVIEW

**Review by Jawshy — DISMISSED** [review]
Thanks for the pull request, @chatainsim. Unfortunately, there are a couple of issues that I'm hoping you could resolve. Please resize the image to exactly 32 by 32 pixels and under 2,500 bytes in size. Saving the image at a bit depth of 8 rather than 32 can help reduce the file size.

**Comment by chatainsim** [comment]
Done

**Comment by nagromc** [comment]
I think the Twitter account should be [@CaissEpargneSAV](https://twitter.com/CaissEpargneSAV) (I know, an 'e' is missing at character 6) rather than [@Caisse_Epargne](https://twitter.com/Caisse_Epargne). The first one is for asking questions (_Nous répondons à vos questions_). The second one is for marketing purposes (_Découvrez l'actualité de la marque Caisse d'Epargne à travers ses multiples engagements !_).

See my (submitted too quickly) [PR](https://github.com/2factorauth/twofactorauth/pull/2996/files#diff-7227dddb57500925d273116121c64aa5R168).

**Comment by chatainsim** [comment]
@nagromc yes it's better I think. I didn't know that they have a 'SAV' account. Good to know.

**Comment by Carlgo11** [comment]
How about this logo instead?
It doesn't contain that white border.

logo

Other than that the PR looks good 😄

**Comment by chatainsim** [comment]
@Carlgo11 thanks, updated

**Review by Carlgo11 — APPROVED** [review]
Looking good! 👍

File: _data/banking.yml [hunk 1]
```diff
@@ -163,6 +163,14 @@ websites:
       phone: Yes
       doc: https://www.mybankofinternet.com/tob/live/usp-core/static/help.html#login_security
 
+    - name: Caisse D'Eparnge
```
Comment (Carlgo11) [inline _data/banking.yml]: On their Facebook and Twitter page they've named themselves Caisse d'Epargne with a lowercase d. I don't speak French so I won't try to request any changes here but if it is a typo here you might want to change that. 😉 
If not then this PR is good to go.

**Comment by chatainsim** [comment]
Updated 👍  Thanks

**Review by nagromc — CHANGES_REQUESTED** [review]
There is a typo: "Epar**gn**e"

**Comment by chatainsim** [comment]
Fixed

**Comment by chatainsim** [comment]
Fixed

**Comment by Carlgo11** [comment]
Now https://github.com/2factorauth/twofactorauth/pull/2977#discussion_r167727169 is reverted. :/

**Comment by chatainsim** [comment]
@carlgo11 what does that mean?

**Comment by stephengroat** [comment]
@chatainsim the `d` has been upper-cased again to `D`

**Comment by chatainsim** [comment]
I'm not able to find any capital D, sorry.

**Comment by nagromc** [comment]
> I'm not able to find any capital D, sorry.

I agree. See the [line 166 of modified file](https://github.com/2factorauth/twofactorauth/pull/2977/files#diff-7227dddb57500925d273116121c64aa5R166) in the PR. I don't see the problem @stephengroat @Carlgo11

**Review by nagromc — APPROVED** [review]

**Review by stephengroat — APPROVED** [review]
thanks for the pr @chatainsim

**Review by Carlgo11 — COMMENTED** [review]
https://github.com/2factorauth/twofactorauth/pull/2977#issuecomment-368823371
Right. And I asked you to change it to uppercase.

**Edit: I fixed it in 14c286878c5919691ea8e2ac7c632208d949a35b**

**Comment by Carlgo11** [comment]
I'll merge it when https://github.com/2factorauth/twofactorauth/pull/2977#pullrequestreview-99906355 is solved.

**Comment by chatainsim** [comment]
Thanks, I was reading too quickly. My bad.

**Comment by fpigerre** [comment]
@Carlgo11 According to convention, the capital letter in this sentence is already the "É" in Épargne. Capitalising the "d" is unnecessary.

This is shown on the Caisse d'Épargne Facebook page as well.

screen shot 2018-04-05 at 13 38 07

**Pull request merged**